packages/core/src/components/divider/divider.tsx

```diff
@@ -0,0 +1,14 @@
+import { Component, h, Prop } from '@stencil/core'
+
+@Component({
+  tag: 'atom-divider',
+  styleUrl: 'divider.scss',
+  shadow: true,
+})
+export class AtomDivider {
+  @Prop() type: 'up' | 'down' = 'down'
+
+  render() {
+    return <div data-type={this.type} class='atom-divider' />
```

Comment:
- we must use `aria-hidden={true}` to improve accessibility
- why are we using `data-` to control styles instead classes?

Reply:
> we must use aria-hidden={true} to improve accessibility

Nice! 
Made: https://github.com/juntossomosmais/atomium/pull/692/commits/249785fcceaede6dfb11b0fa5dba84fd7b32a64f

> why are we using data- to control styles instead classes?

Good observation! It makes more sense to use classes to maintain readability and follow the standard convention. I refactored to replace data types with classes.

Made: https://github.com/juntossomosmais/atomium/pull/692/commits/8f03d80a999516b2e709a166844dedd567b6f713